### PR TITLE
Remove last traces of OPNVKarte layer

### PR DESCRIPTION
**Pull request by tomhughes**
The layer was removed in 75a4d406b9e8471c0fe0ebf2a47d47f0ae36289a but some traces were missed.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -2382,8 +2382,6 @@ en:
           private: "Private access"
           destination: "Destination access"
           construction: "Roads under construction"
-          bus_stop: "Bus stop"
```
[the remark this answers is not in the document]
Comment (tomhughes) [reply config/locales/en.yml:2385]: Nothing else is currently using it but we certainly can keep it.

[the remark this answers is not in the document]
Comment (tomhughes) [reply config/locales/en.yml:2385]: Done.

File: app/assets/javascripts/embed.js.erb [hunk 2]
```diff
@@ -45,8 +45,6 @@ window.onload = function () {
     new L.OSM.CycleMap(thunderforestOptions).addTo(map);
   } else if (args.layer === "transportmap") {
     new L.OSM.TransportMap(thunderforestOptions).addTo(map);
-  } else if (args.layer === "opnvkarte") {
-    new L.OSM.OPNVKarte().addTo(map);
```
[the remark this answers is not in the document]
Comment (AntonKhorev) [reply app/assets/javascripts/embed.js.erb:49]: Given that mapquest also falls back, all those conditions could be *if cyclosm show cyclosm; if cyclemap show cyclemap; ... ; else show mapnik*.

[the remark this answers is not in the document]
Comment (tomhughes) [reply app/assets/javascripts/embed.js.erb:49]: I've changed it to put mapnik last now, and use that for any unknown layer name.

**Comment by AntonKhorev** [comment]
Merged, thanks.

**Pull request merged**